charts/horizon/values.yaml

```diff
@@ -49,7 +49,8 @@ config:
       server: http://el-horizon-listener:8080
       namespace: "{{ .Release.Namespace }}"
       kubeconfig: ""
-      s3:
+      logStorage:
```

Comment:
If you remove minio, Harbor need to do some modifition because it use minio as the backend storage layer.

Reply:
minio and harbor will be installed together in full mode and removed together in minimal mode